ReactiveCocoa/UIKit/UIGestureRecognizer.swift

```diff
@@ -0,0 +1,25 @@
+import ReactiveSwift
+import UIKit
+import enum Result.NoError
+
+extension Reactive where Base: UIGestureRecognizer {
+	
+	/// Create a signal which sends a `next` event for each gesture event
+	///
+	/// - returns:
+	///   A trigger signal.
+	
+	public var recognised: Signal<(), NoError> {
```

Comment:
The name still needs bikeshedding though. `recognised` has an established meaning in the context of `UIGestureRecognizer`.